client/packages/invoices/src/Prescriptions/DetailView/PrescriptionLineEdit/PrescriptionLineEditForm.tsx

```diff
@@ -145,8 +145,8 @@ export const PrescriptionLineEditForm: React.FC<
     updateIssueQuantity(allocatedQuantity);
   };
 
-  const handleIssueQuantityChange = (quantity?: number) => {
-    if (quantity === undefined) return;
+  const handleIssueQuantityChange = (inputQuantity?: number) => {
+    const quantity = inputQuantity === undefined ? 0 : inputQuantity;
```

Comment:
The actual changes that fixes the original issue. Makes it always set a value -- defaults to `0` if undefined.